napari/layers/labels/labels.py

```diff
@@ -461,7 +466,7 @@ def data(self, data):
         data = self._ensure_int_labels(data)
         self._data = data
         self._update_dims()
-        self.events.data(value=self.data)
+        # self.events.data(value=self.data)
```

Comment:
```suggestion
        # self.events.data(value=self.data)
```

Could you explain why this was commented out?

Reply:
Thank you for pointing out! It is just a typo, I commented it accidentally. I fixed it.